aspnetcore/fundamentals/websockets.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: WebSockets support in ASP.NET Core
-author: rick-anderson
+author: wadepickett
 description: Learn how to get started with WebSockets in ASP.NET Core.
 monikerRange: '>= aspnetcore-3.1'
-ms.author: riande
```

Comment:
Why replace the MS author? That can act as the backup author.

Reply:
@Rick-Anderson, per doc manager, (as stated above).  I'm supposed to own these entirely going forward.  Both author and ms.author settings.